steps/ostree.py

```diff
@@ -4,14 +4,16 @@
 from behave import *
 
 
-@then(u'undeploy the unselected deployment')
-def step_impl(context):
+def get_ostree_admin_status(context):
+    '''get ostree admin status'''
     ostree_result = context.remote_cmd(cmd='command',
                                        module_args='ostree admin status')
+    return ostree_result[0]['stdout']
 
-    assert ostree_result, "Can not get ostree status"
 
-    ostree_status = ostree_result[0]['stdout']
+@then(u'undeploy the unselected deployment')
+def step_impl(context):
+    ostree_status = get_ostree_admin_status(context)
```

Comment:
I think you should check that `ostree_status` has valid value before trying to use it.